co-simulation tool/README.md

```diff
@@ -0,0 +1,250 @@
+
+
+# Carla-Sumo-Mosaic Co-Simulation Tool
+
+This co-simulation tool is used to run [CARLA simulator](http://carla.org/) and [SUMO simulator](https://www.eclipse.org/sumo/) with [Eclipse MOSAIC framework](https://github.com/eclipse/mosaic). The co-simulation tool can simulate both co-simulation scenarios and original Eclipse MOSAIC scenarios on both Windows systems and Linux systems.  
+
+## Installation of Co-simulation Tool
+
+**a.**   **Executable bundle file**
+
+Download the CARLA-SUMO-MOSAIC-1.0.zip bundle and extract it to an arbitrary path. 
+
+**b.**   **Installation from source code**
+
+Download the co-simulation tool code and refer to the Eclipse-MOSAIC build instruction in this [link]( https://github.com/eclipse/mosaic) .
+
+***Note:*** **The source code is in the folder of co-simulation tool rather than the root folder.** To build it, users should first go to the folder of co-simulation tool and then build it.
+
+The installation path is referenced as <cosimulation-root>. After the co-simulation tool is installed from source code, the zipped bridge folder needs to be extracted to <cosimulation-root>.
```

Comment:
1. <cosimulation-root> does not display on the page. Please correct it. 
2. Please use the name of zipped bridge folder instead of "the zipped bridge folder". The name will be clearer for users.